apps/src/blockTooltips/DropletBlockTooltip.html.ejs

```diff
@@ -0,0 +1,7 @@
+<div class="function-name">
+  <%= functionName %>(<% for (var i = 0; i < parameters.length; i++) { -%><%- parameters[i] -%><% if (i < parameters.length - 1) { -%>, <% } -%><% } -%>)
```

Comment:
I'll take another stab at this, I had trouble finding a way to make it well-formatted with new lines without introducing extra spaces between strings. I played with incantations of `<%-`s `-%>`s, etc.